src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4359,6 +4359,47 @@ def test_ps_s3_multiple_topics_notification():
     http_server.close()
 
 
+@attr('kafka_test')
```

Comment:
to verify getting the topic list, you do not need to actually send them.
so, better to mark that as a `basic_test`.